src/components/squareComponent/square.spec.js

```diff
@@ -14,4 +14,55 @@ describe('Square Component', () => {
     ReactDOM.unmountComponentAtNode(gameSpaceDiv);
   });
 
+  it('tests the square is empty before the user has placed a marker on the square', () => {
```

Comment:
It's a bit hard for me to see what to focus of a given test is in this file. I am personally a fan of the "Arrange/Act/Assert" pattern for organizing an individual unit test.

The short version is that each test will have exactly two (or in some cases just one) newline separating the "arrange" from the "act" and the "act" from the "assert". Here are some blogs going into more detail:
 * https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80
     * Note: I would _not_ have actual comments in the test describing each section.
 * http://wiki.c2.com/?ArrangeActAssert

Reply:
Very informative! I modified all my spec files to mimic the AAA pattern for my unit tests.